src/Integration.Vsix.UnitTests/SonarLintDaemon/CFamilyTest.cs

```diff
@@ -148,24 +148,26 @@ public void PlatformName()
         [TestMethod]
         public void PlatformToolset()
         {
-            CFamily.FileConfig.ConvertPlatformToolset("v141").Should().Be("19.10.00");
```

Comment:
I have reordered the test so that new platforms are added to the bottom of the test.